File: unitethiscity.com/App_Code/SummaryStats.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

/// <summary>
/// Summary description for SummaryStats
/// </summary>
public class SummaryStats
{
    public string Name { get; set; }
    public string Link { get; set; }
    public int Today { get; set; }
    public int PastWeek { get; set; }
    public int ThisPeriod { get; set; }
    public int LastPeriod { get; set; }
    public int AllTime { get; set; }

    public SummaryStats(string name = "")
    {
        Name = name;
        Today = 0;
        PastWeek = 0;
        ThisPeriod = 0;
        LastPeriod = 0;
        AllTime = 0;
    }
}

File: android/unitethiscity/uniteThisCity/src/main/java/com/unitethiscity/data/SummaryStats.java
package com.unitethiscity.data;

/**
 * Created by Don on 2/16/2016.
 */
public class SummaryStats {

    String Name;
    String Link;
    int Today;
    int PastWeek;
    int ThisPeriod;
    int LastPeriod;
    int AllTime;

    public SummaryStats(String name, String link, int today, int pastWeek, int thisPeriod, int lastPeriod, int allTime) {
        Name = name;
        Link = link;
        Today = today;
        PastWeek = pastWeek;
        ThisPeriod = thisPeriod;
        LastPeriod = lastPeriod;
        AllTime = allTime;
    }

    public String getName() {
        return Name;
    }

    public String getLink() {
        return Link;
    }

    public int getToday() {
        return Today;
    }

    public int getPastWeek() {
        return PastWeek;
    }

    public int getThisPeriod() {
        return ThisPeriod;
    }

    public int getLastPeriod() {
        return LastPeriod;
    }

    public int getAllTime() {
        return AllTime;
    }
}
